inspector/src/components/actionTabs/tabs/propertyGrids/animations/timeline.tsx

```diff
@@ -346,11 +350,17 @@ export class Timeline extends React.Component<
       const frameChange = (resizePercentage * this.state.end) / 100;
       const framesTo = Math.round(frameChange);
 
-      this.setState({
-        end: framesTo,
-        scrollWidth: this.calculateScrollWidth(this.state.start, framesTo),
-        selectionLength: this.range(this.state.start, framesTo),
-      });
+      if (framesTo <= this.state.start + 20) {
+        console.log('broke');

```

Comment:
yes I am, but no need to show it to the world! :-)

Is this if needed?

Reply:
I still have a flickering scrollbar bug and I'm using it to track it. I can remove it and add it to my notes :)

Reply:
I just prefer not to commit any debug code into a system used in production